plugins/synaptics-rmi/fu-synaptics-rmi-common.h

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2012-2014 Andrew Duggan
+ * Copyright (C) 2012-2019 Synaptics Inc.
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+#define FU_SYNAPTICS_RMI_DEVICE_VID			0x06cb
```

Comment:
Given only needed in one file, maybe move it over to the source.

Reply:
Removed